doonut-core/src/main/java/com/doonutmate/doonut/member/repository/MemberRepository.java

```diff
@@ -17,6 +18,24 @@ public interface MemberRepository extends JpaRepository<MemberEntity, Long> {
             """)
     Optional<MemberEntity> findByOauthId(String oauthId);
 
+    @Query("""
+                SELECT member.fcmToken
+                FROM MemberEntity member
+                WHERE member.deleted= false AND member.serviceAlarm = true AND member.fcmToken IS NOT NULL
+            """)
+    List<String> findByFcmTokenList();
+
+    @Query("""
+                SELECT member.name
+                FROM MemberEntity member
+                WHERE member.deleted = false AND member.fcmToken = :fcmToken
+            """)
+    String findByName(String fcmToken);
+
+    @Modifying(clearAutomatically = true)
+    @Query("UPDATE MemberEntity m SET m.fcmToken = :deviceToken WHERE  m.id = :memberId")
```

Comment:
WHERE 절에 불필요한 띄어쓰기가 있네용
```suggestion
    @Query("UPDATE MemberEntity m SET m.fcmToken = :deviceToken WHERE m.id = :memberId")
```

Reply:
정말 그러네요 🥲
수정했습니다.